docs/Project-Lifecycle.md

```diff
@@ -324,6 +325,20 @@ Then you are able to review the tests results in [Test results page](http://comm
 
 To test with Paypal, you need own Sandbox Buyer credentials via a developer account.
 
+### Klarna Testing notes
+
+Klarna has very strict requirements to the payment details. For testing purpose one should use
+[special Klarna test credentials](https://developers.klarna.com/en/de/kpm/test-credentials).
+
+These tests require next CTP settings:
+
+  * `gender`, `ip` and `birthday` custom fields in _payment-INVOICE-KLARNA_ custom type
+  * German language, country and prices must be active.
+  * 19% German tax is active
```

Comment:
of course we could extend our integration tests to be self-sufficient and self-configured, but this requires additional task/time, which is not a part of this task